migrations/versions/7301d5130c3a_.py

```diff
@@ -0,0 +1,47 @@
+"""Fixing type in EventHandlerOption table from urlendcode to urlencoded
+
+Revision ID: 7301d5130c3a
+Revises: 69e7817b9863
+Create Date: 2024-11-29 12:03:46.090125
+
+"""
+from alembic import op
+from sqlalchemy import orm
+from privacyidea.models import EventHandlerOption
+
+# revision identifiers, used by Alembic.
+revision = '7301d5130c3a'
+down_revision = '69e7817b9863'
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+    try:
+        for row in session.query(EventHandlerOption).filter(EventHandlerOption.Value == 'urlendcode'):
```

Comment:
i think we should probably go with something like this:
```
op.execute("UPDATE eventhandleroption SET Value='urlencoded' WHERE Value='urlendcode'")
```
(See https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.execute)
I'll have to check if this works with all currently supported databases but this is regular SQL...

Reply:
We could also use a rudimentary table definition or reflect it from the actual database and use SQLAlchemy constructs here.